zeth-contracts/contracts/BaseMixer.sol

```diff
@@ -244,6 +241,25 @@ contract BaseMixer is MerkleTreeMiMC7, ERC223ReceivingContract {
         hsig = bytes32(high_bits + uint(hsig_bytes));
     }
 
+    // This function is used to reassemble the commitment given the commitment
+    // index [0, jsOut[ and the primary_inputs. To do so, we extract the
+    // remaining bits of the commitment from the residual field element(S) and
+    // combine them with the commitment field element.
+    function assemble_commitment(
```

Comment:
Please follow [NatSpec](https://solidity.readthedocs.io/en/v0.6.4/natspec-format.html) for "documentation comments"

Reply:
Shouldn't this be a separate issue (if we want to update all comments)?

Reply:
let's do that yes, that works for me

Reply:
Issue #192 created.